includes/options/views/class-amp-setup-wizard-submenu-page.php

```diff
@@ -148,6 +154,7 @@ public function enqueue_assets( $hook_suffix ) {
 			],
 			'OPTIONS_REST_ENDPOINT'              => rest_url( 'amp/v1/options' ),
 			'READER_THEMES_REST_ENDPOINT'        => rest_url( 'amp/v1/reader-themes' ),
+			'PREVIEW_PERMALINK'                  => amp_admin_get_preview_permalink(),
```

Comment:
See comment above about how this should ideally be returned by the REST API after saving the mode.

Reply:
```suggestion
```